core/src/components/picker-column-option/picker-column-option.tsx

```diff
@@ -0,0 +1,55 @@
+import type { ComponentInterface } from '@stencil/core';
+import { Component, Element, Host, Prop, h } from '@stencil/core';
+import type { Attributes } from '@utils/helpers';
+import { inheritAriaAttributes } from '@utils/helpers';
+
+import { getIonMode } from '../../global/ionic-global';
+
+@Component({
+  tag: 'ion-picker-column-option',
+  shadow: true,
+})
+export class PickerColumnOption implements ComponentInterface {
+  private optionId = `ion-picker-opt-${pickerOptionIds++}`;
+
+  private inheritedAttributes: Attributes = {};
+
+  @Element() el!: HTMLElement;
+
+  /**
+   * If `true`, the user cannot interact with the select option.
+   */
+  @Prop() disabled = false;
+
+  /**
+   * The text value of the option.
+   */
+  @Prop() value?: any | null;
+
+  componentWillLoad() {
+    this.inheritedAttributes = inheritAriaAttributes(this.el);
```

Comment:
As per the ticket, we should be taking advantage of Stencil's watch decorator for attributes: https://stenciljs.com/docs/reactive-data#watching-native-html-attributes

Reply:
Thank you both for the deeper explanations. I now have a better idea on how this stuff works.